src/components/Dashboard/TodoList.vue

```diff
@@ -1272,6 +1278,45 @@ export default class TodoList extends Mixins(
     }
   }
 
+  private async loadSpecialResolution (task: ApiTaskIF): Promise<void> {
+    const filing = task.task.filing
+    const header = filing.header
+    const business = filing.business
+    const specialResolution = filing.specialResolution
+
+    if (header && specialResolution) {
+      if (!this.isGoodStanding && this.isStatusDraft(header)) {
+        task.enabled = false
```

Comment:
Well, you _shouldn't_ change the original data, but I know it's not used for anything else so not a big deal.

Reply:
I borrowed this from Alteration

Reply:
Doh. Just leave it then 😋